From cb562a383229074b44db6998b85572d2abe7c363 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?An=C3=ADbal=20Lim=C3=B3n?= <anibal.limon@linaro.org>
Date: Fri, 24 Nov 2017 10:31:55 -0600
Subject: [PATCH] tests/egl/egl-context-priority.c: Use
 piglit_egl_get_default_display
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The entry point is not guaranteed to exist, so use the
piglit_egl_get_default_display() helper which does the correct thing.

Signed-off-by: Aníbal Limón <anibal.limon@linaro.org>
Signed-off-by: Daniel Díaz <daniel.diaz@linaro.org>
Reviewed-by: Tapani Pälli <tapani.palli@intel.com>
Reviewed-by: Emil Velikov <emil.velikov@collabora.com>

Upstream-Status: Backport

https://cgit.freedesktop.org/piglit/commit/?id=cb562a3832
---
 tests/egl/egl-context-priority.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/tests/egl/egl-context-priority.c b/tests/egl/egl-context-priority.c
index 7f26fc614..9590ccae3 100644
--- a/tests/egl/egl-context-priority.c
+++ b/tests/egl/egl-context-priority.c
@@ -99,8 +99,7 @@ piglit_init(int argc, char **argv)
 	if (!strstr(exts, "EGL_MESA_platform_surfaceless"))
 		piglit_report_result(PIGLIT_SKIP);
 
-	dpy = eglGetPlatformDisplay(EGL_PLATFORM_SURFACELESS_MESA,
-				    EGL_DEFAULT_DISPLAY, NULL);
+	dpy = piglit_egl_get_default_display(EGL_PLATFORM_SURFACELESS_MESA);
 
 	ok = eglInitialize(dpy, &major, &minor);
 	if (!ok) {
-- 
2.11.0

